exporter/zipkin/example_test.go

```diff
@@ -17,24 +17,17 @@ package zipkin_test
 import (
 	"log"
 
-	openzipkin "github.com/openzipkin/zipkin-go"
-	"github.com/openzipkin/zipkin-go/reporter/http"
 	"go.opencensus.io/exporter/zipkin"
 	"go.opencensus.io/trace"
 )
 
 func Example() {
-	// import (
-	//     openzipkin "github.com/openzipkin/zipkin-go"
-	//     "github.com/openzipkin/zipkin-go/reporter/http"
-	//     "go.opencensus.io/exporter/trace/zipkin"
-	// )
-
-	localEndpoint, err := openzipkin.NewEndpoint("server", "192.168.1.5:5454")
+	exporter, err := zipkin.NewExporterWithOptions(&zipkin.Options{
+		EndpointHostPort: "192.168.1.5:5454",
+		ReporterURI:      "localhost:9411/api/v2/spans",
```

Comment:
s/ReporterURI/Reporter

Your example misses the schema.

Reply:
ACK about missing the scheme, but Reporter on it's own doesn't convey that it's a URI. Should I change it to Reporter and then add a comment that Reporter is the URI? Do you think that'll be sufficient?

Reply:
What about ReporterEndpoint? A URL or URI prefix is very specific.

We use Endpoint for Jaeger: https://godoc.org/go.opencensus.io/exporter/jaeger#Options.